docs/docs-content/user-management/saml-sso/palette-sso-with-custom-ca.md

```diff
@@ -0,0 +1,52 @@
+---
+sidebar_label: "Palette SSO with Custom CA"
+title: "Enable SSO with a Custom Certificate Authority"
+description: "Learn how to set up Palette SSO with Microsoft Entra ID"
+hide_table_of_contents: false
+sidebar_position: 130
+hiddenFromNav: false
+tags: ["user-management", "oidc-sso", "custom-ca", "rbac"]
+---
+
+A custom Certificate Authority (CA) refers to a certificate authority that is not part of the standard public CA
+ecosystem but is instead managed internally within an organization. This guide explains how you can enable Palette SSO
+with a custom CA certificate.
+
+## Prerequisites
+
+- Palette account with Tenant Admin access.
+
+- Existing SSO configuration with an SSO provider supported in Palette. Refer to [SAML and OIDC SSO Setup](saml-sso.md)
+  for links to provider-specific guides.
+
+- Custom CA x509 certificate in the Privacy-Enhanced Mail (PEM) format issued by the SSO provider configured in Palette.
+
```

Comment:
Let's add the prereq that communication between Palette and the Identity Provider must be over HTTPS. In other words, we don't support non-encrypted network traffic.

![CleanShot 2024-07-11 at 10 08 12](https://github.com/spectrocloud/librarium/assets/29551334/95086183-cd70-4503-b107-e83307a2561e)